.github/workflows/android.yml

```diff
@@ -0,0 +1,132 @@
+name: Dart SDK Android Build
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "dart-sdk@*"
+    paths:
+      - "dart-sdk/rust/**"
+      - "eppo_core/**"
+      - ".github/workflows/android.yml"
+  pull_request:
+    paths:
+      - "dart-sdk/rust/**"
+      - "eppo_core/**"
+      - ".github/workflows/android.yml"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+env:
+  CI: true
+
+jobs:
+  build-android:
+    name: Build Android (${{ matrix.target }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          [aarch64-linux-android, x86_64-linux-android, armv7-linux-androideabi]
+        include:
+          - target: aarch64-linux-android
+            android_abi: arm64-v8a
+          - target: x86_64-linux-android
+            android_abi: x86_64
+          - target: armv7-linux-androideabi
+            android_abi: armeabi-v7a
```

Comment:
https://developer.android.com/ndk/guides/abis

Could you try to add `x86` 32-bit and see if it compiles? That will cover all our bases.

Reply:
This covers all major platforms - really awesome to be able to deliver this that feels every bit as comprehensive as native code